src/firstvoices/backend/tests/utils.py

```diff
@@ -0,0 +1,8 @@
+import random
+import string
+
+
+def generate_string(length):
+    """Function to generate string of ascii characters (both upper and lower case) for the given length."""
+    letters = string.ascii_letters
+    return "".join(random.choice(letters) for _ in range(length))  # NOSONAR
```

Comment:
Thank you. I have added this, but we will still have to keep the NOSONAR in there, since sonar does not like the random library.
The alternative is to use the secrets library instead, one similar method I found was generating url safe tokens of a given length using this library. 
https://docs.python.org/3/library/secrets.html#generating-tokens

I believe using random here in tests is okay but can switch it if required.

Reply:
Random in tests is fine with me-- this is probably the most common thing I mark as "won't fix" in sonar